<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>775</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1036</width>
    <height>775</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>КГ Лабораторная работа №3</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QGraphicsView" name="canvas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>332</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отменить действие</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="maximumSize">
         <size>
          <width>1600043</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отменить все</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="lineColorButton">
            <property name="text">
             <string>Цвет линий</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lineColorBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="bgColorButton">
            <property name="text">
             <string>Цвет фона</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bgColorBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Метод</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodBox">
        <item>
         <property name="text">
          <string>ЦДА</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхем (целые)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхем (вещественные)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхем (сглаживание)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ву</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Добавить прямую</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineXStartInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>X начала</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineYStartInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Y начала</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineXEndInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>X конца</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineYEndInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Y конца</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addLineButton">
        <property name="maximumSize">
         <size>
          <width>1600043</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Нарисовать прямую</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Добавить спектр</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="spectreXCentreInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>X центра</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="spectreYCentreInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Y центра</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="spectreRadiusInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Радиус</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="spectreAngleInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>454636</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Угол</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addSpectreButton">
        <property name="text">
         <string>Нарисовать спектр</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Показать графики</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="showTimeButton">
          <property name="text">
           <string>Время</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showStepsButton">
          <property name="text">
           <string>Ступенчатость</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Расширенные</string>
     </property>
     <widget class="QMenu" name="menu_4">
      <property name="title">
       <string>Дополнительные</string>
      </property>
      <widget class="QMenu" name="menu_5">
       <property name="title">
        <string>Параметры</string>
       </property>
       <widget class="QMenu" name="menu_6">
        <property name="title">
         <string>Профессиональные</string>
        </property>
        <widget class="QMenu" name="menu_7">
         <property name="title">
          <string>Возможности</string>
         </property>
         <widget class="QMenu" name="menu1">
          <property name="title">
           <string>1</string>
          </property>
          <widget class="QMenu" name="menu2">
           <property name="title">
            <string>2</string>
           </property>
           <addaction name="action_6"/>
          </widget>
          <addaction name="menu2"/>
         </widget>
         <addaction name="menu1"/>
        </widget>
        <addaction name="menu_7"/>
       </widget>
       <addaction name="menu_6"/>
      </widget>
      <addaction name="menu_5"/>
     </widget>
     <addaction name="menu_4"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="programInfoAction"/>
    <addaction name="authorInfoAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="programInfoAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="authorInfoAction">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>мрлиыаорм</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
